ext/intl/calendar/calendar.stub.php

```diff
@@ -1,154 +1,283 @@
 <?php
 
+/** @generate-function-entries */
+
 class IntlCalendar
 {
     private function __construct() {}
 
     /**
      * @param IntlTimeZone|DateTimeZone|string|null $timeZone
-     * @return IntlCalendar|null
+     * @return static|null
```

Comment:
I changed the return type to `static` because `IntlGregorianCalendar` inherits it

Reply:
I don't think this is right, as it will only return `IntlCalendar` or `IntlGregorianCalendar`, but those can be further subclassed. I also don't think that the returned class actually depends what you invoke the method on, i.e. `IntlGregorianCalendar::createInstance()` may return an `IntlCalendar` if you specify a non-gregorian calendar.

Reply:
Thank you for the explanation! I changed the return type to `IntlCalendar|IntlGregorianCalendar|null`.